lib/reply.js

```diff
@@ -33,46 +33,31 @@ Reply.prototype.send = function (payload) {
     this.res.log.warn(new Error('Reply already sent'))
     return
   }
-
   this.sent = true
 
   if (payload instanceof Error || this._isError === true) {
     handleError(this, payload, onSendHook)
     return
   }
 
-  if (payload === undefined) {
```

Comment:
Why is this short circuit removed?

Reply:
It's changed to:
```javascript
if (payload !== undefined)
```
So that `onSendHook(this, payload)` is called just once